starter-code/src/clue.js

```diff
@@ -1,94 +1,157 @@
+};
+
+var mrMustard = {
+  first_name: "Jack",
+  last_name: "Mustard",
+  color: "yellow",
+  description:
+    "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player"
+};
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
-
-// Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+var rope = { weapon: "rope", weight: 10 };
+var knife = { weapon: "knife", weight: 8 };
+var candlesti = { weapon: "candlesti", weight: 2 };
+var dumbbel = { weapon: "dumbbel", weight: 30 };
+var poison = { weapon: "poison", weight: 2 };
+var axe = { weapon: "axe", weight: 15 };
+var bat = { weapon: "bat", weight: 13 };
+var trophy = { weapon: "trophy", weight: 25 };
+var pistol = { weapon: "pistol", weight: 20 };
+
+// Roomsame: Dinning Room
+
+var conservatory = { room: "Conservatory" };
+var kitchen = { room: "Kitchen" };
+var study = { room: "Study" };
+var library = { room: "Library" };
+var billiardRoom = { room: "Billiard Room" };
+var lounge = { room: "Lounge" };
+var hall = { room: "Hall" };
+var spa = { room: "Spa" };
+var livingRoom = { room: "Living Room" };
+var observatory = { room: "Observatory" };
+var theater = { room: "Theater" };
+var guestHouse = { room: "Guest House" };
+var patio = { room: "Patio" };
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [
+  mrGreen,
+  mrMustard,
+  mrsPeacock,
+  missScarlet,
+  profPlum,
+  drOrchid
+];
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio
+];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [
+  rope,
+  knife,
+  candlesti,
+  dumbbel,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+
+function randomSelector(array) {
+  return array[Math.floor(Math.random() * (array.length - 0)) + 0];
+}
+
+var misteryEnvelope = [];
+
+function pickMistery() {
+  misteryEnvelope.push(randomSelector(charactersArray));
+  misteryEnvelope.push(randomSelector(weaponsArray));
+  misteryEnvelope.push(randomSelector(roomsArray));
+  return misteryEnvelope;
+}
+
+function revealMistery(array) {
+  return console.log(
+    array[0].first_name +
+      " " +
+      array[0].last_name +
+      " killed Mr.Boddy using the " +
+      array[1].weapon +
+      " in the " +
```

Comment:
Jasmine esperaba la propiedad name de los objetos Weapon y Room ` " killed Mr.Boddy using the " +
    array[1].name +
    " in the " +
    array[2].name + "!!!!"`